src/components/pages/AdminDashboard/DevTools/dayData.js

```diff
@@ -1,6 +1,7 @@
 export const dayData = [
   {
     dayID: 1,
+    dayOfWeekIndex: 6, //used for findNextDayOfWeek
```

Comment:
I like this setup. I'm assuming these are just pointers to the next day but might be confusing to some people if you start Saturday with an index of 6 when its the first day of play.

Reply:
I agree with Yinusa - it might get confusing with the index of the day, but everything else looks great!

Reply:
It has to be numbered this way for the algorithm to function right. 

I'm not sure if index of day is the best name, I can change the name. Maybe dayOfWeekNumber?